src/main/java/vazkii/botania/common/block/tile/TileTerraPlate.java

```diff
@@ -61,9 +58,9 @@
 					}
 			},
 			'P', ModBlocks.terraPlate,
-			'R', ModBlocks.livingrock,
-			'0', ModBlocks.livingrock,
-			'L', Blocks.LAPIS_BLOCK
+			'R', "#botania:livingrock",
+			'0', "#botania:livingrock",
+			'L', "#forge:storage_blocks/lapis"
```

Comment:
What this should use is something like
`StateMatcher.fromPredicate(ModBlocks.livingrock, state -> state.getBlock().isIn(ModTags.Blocks.LIVINGROCK))`
like the mana enchanter is doing for flowers.